reference-infrastructure/storage/nfs/nfs-server.yaml

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
That looks neat, do we have stable (ish) green builds on Bitbucket with that setup?